IDR/production/0.collect_screen_metadata.py

```diff
@@ -0,0 +1,81 @@
+from asyncio import subprocess
+import pathlib
+import sys
+import requests
+import subprocess
+import pandas as pd
+from utils.args import *
```

Comment:
I generally like to avoid importing wildcards, especially for custom functions. An outsider looking at this code can become confused if they see a presumably-undefined function

Reply:
Removed unused imports and specified `utils.args` import as `collect_screen_metadata_parser`

Added in 118ae1a01a3485c7e954f25a5c9f2659a50ab80e.